CMakeLists.txt

```diff
@@ -71,6 +71,9 @@ OPTION (ALLOW_OLD_GETTEXT "allow to configure build with a gettext version older
 
 # These are also settable from the command line in a similar way.
 
+# Use gsettings-desktop-schema for a better integration with GNOME
+SET(COMPILE_SCHEMA ON CACHE BOOL "Compile the GSettings schema")
```

Comment:
If there is not already some convention, I would name the option COMPILE_GSCHEMA
to have 2 letters distance from SCHEME stuff

Reply:
It's a good idea.